polygeist/test/polygeist-opt/memoryAccess.mlir

```diff
@@ -7,43 +7,104 @@
 !sycl_accessor_1_f32_rw_gb = !sycl.accessor<[1, f32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl_id_1, !sycl_range_1, !sycl_range_1)>, !llvm.struct<(memref<?xf32, 1>)>)>
 !sycl_accessor_2_f32_rw_gb = !sycl.accessor<[2, f32, read_write, global_buffer], (!sycl.accessor_impl_device<[2], (!sycl_id_2, !sycl_range_2, !sycl_range_2)>, !llvm.struct<(memref<?xf32, 1>)>)>
 
-// COM: Test 1-dim accessor memory access yielding an identity matrix.
+// COM: Test 1-dim accessor memory access.
+//      The underlying value of the accessor subscript is the loop IV.
 // CHECK-LABEL: test_tag: test1_load1
 // CHECK: matrix:
 // CHECK-NEXT: 1
 func.func @test1(%acc : memref<?x!sycl_accessor_1_f32_rw_gb, 4>) {
```

Comment:
This test has been modified to check that we can determine the multipler of the loop IV in the presence of load/store operations that copy the result of the sycl.constructor operation (lines 25,26).